udata_front/theme/gouvfr/assets/js/pages/PublishingForm/Step3UpdateFileMetadata.vue

```diff
@@ -0,0 +1,376 @@
+                </template>
+              </i18n-t>
+            </Accordion>
+            <Accordion
+              :title= "$t('What is a Mime type')"
+              :id="whatIsAMimeTypeAccordionId"
+              :state="state.mime"
+            >
+              {{ $t("") }}
+            </Accordion>
+          </AccordionGroup>
+      </Sidemenu>
+      <div class="fr-col-12 fr-col-md-7">
+        <Container>
+          <fieldset class="fr-fieldset min-width-0" aria-labelledby="description-legend">
+            <legend class="fr-fieldset__legend" id="description-legend">
+              <h2 class="subtitle subtitle--uppercase fr-mb-3v">
+                {{ $t("File metadata") }}
+              </h2>
+            </legend>
+            <div class="fr-fieldset__element">
+              <FileCard
+                class="fr-mb-3v"
+                :filename="file.file?.name"
+                :filesize="file.filesize"
+                :format="file.format"
+                :lastModified="file.file?.lastModified"
+                :missingMetadata="v$.$invalid"
+                :title="file.title || file.file?.name || ''"
+                :hideActions="true"
+              />
+            </div>
+            <LinkedToAccordion
+              v-if="isRemote"
+              class="fr-fieldset__element min-width-0"
+              :accordion="chooseTheCorrectLinkAccordionId"
+              @blur="v$.url.$touch"
+            >
+              <InputGroup
+                :label="$t('Link url')"
+                :required="true"
+                v-model="file.url"
+                :hasError="stateHasError('url')"
+                :errorText="getErrorText('url')"
+              />
+            </LinkedToAccordion>
+            <LinkedToAccordion
+              class="fr-fieldset__element min-width-0"
+              :accordion="nameAFileAccordionId"
+              @blur="v$.title.$touch"
+            >
+              <InputGroup
+                :label="fileTitle"
+                :required="true"
+                v-model="file.title"
+                :hasError="stateHasError('title')"
+                :errorText="getErrorText('title')"
+              />
+            </LinkedToAccordion>
+            <LinkedToAccordion
+              class="fr-fieldset__element min-width-0"
+              :accordion="chooseTheRightTypeOfFileAccordionId"
+              @blur="v$.type.$touch"
+            >
+              <SelectGroup
+                :label="$t('Type')"
+                :required="true"
+                v-model="file.type"
+                :hasError="stateHasError('type')"
+                :errorText="getErrorText('type')"
+                :options="fileTypes"
+              />
+            </LinkedToAccordion>
+            <LinkedToAccordion
+              class="fr-fieldset__element min-width-0"
+              :accordion="chooseTheCorrectFormatAccordionId"
+              @blur="v$.format.$touch"
+            >
+              <MultiSelect
```

Comment:
If format is missing, we can't click on `Next`, but it doesn't show in red as missing

Reply:
Fixed